docs/README.md

```diff
@@ -12,6 +12,7 @@
   * [Git (git)](/docs/Options.md#git-git)
     * [Git branch (git_branch)](/docs/Options.md#git-branch-gitbranch)
     * [Git status (git_status)](/docs/Options.md#git-status-gitstatus)
+    * [Git commit time(git_committime)](/docs/Options.md#git-git-commit-time)
```

Comment:
Since this section is not part of the `git` section, you should not nest it under `git` section in documentation too.